contracts/forwarding/IForwarder.sol

```diff
@@ -0,0 +1,14 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity ^0.4.24;
+
+
+interface IForwarder {
+    function isForwarder() external pure returns (bool);
+
+    function canForward(address sender, bytes evmCallScript) external view returns (bool);
+
+    function forward(bytes evmCallScript) external;
```

Comment:
We missed this a long time ago, but we should rename all the `bytes` arguments to `evmScript` instead (a calls script is a specific type of EVM Script in our lingo).